index.html

```diff
@@ -66,5 +71,162 @@ <h2>I’m a software developer</h2>
         <img src="images/icons/down-arrow-icon.svg" alt="Scroll Down Button" />
       </button>
     </section>
+
+    <section class="project-list">
+      <div class="projects-header">
+        <h1 class="header-text">My Recent Works</h1>
```

Comment:
Generally, it is best practice to use only one [h1] tag per HTML page. You can use as many of the other ones as needed.

Reply:
Oh okay